modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -639,6 +640,8 @@ private ObjectDefinition _toObjectDefinition(
 				active = objectDefinition.isActive();
 				dateCreated = objectDefinition.getCreateDate();
 				dateModified = objectDefinition.getModifiedDate();
+				defaultLanguageId = _localization.getDefaultLanguageId(
+					objectDefinition.getLabel());
```

Comment:
@marcelabc we can forward this now. But can you send a follow-up PR creating the column defaultLanguadeId for ObjectDefinition? thx